src/routes/projects/showcase/+page.ts

```
@@ -0,0 +1,16 @@
+import type { PageLoad } from './$types';
+
+type EnhancedImg = typeof import('*.png?enhanced').default;
+
+export const load = (({ data }) => {
+	const images: Record<string, { default: EnhancedImg }> = import.meta.glob('$contents/projects/showcase/*.{avif,gif,heif,jpeg,jpg,png,tiff,webp,svg}', { eager: true, query: { enhanced: true } });
+	const projects = data.projects.map((project) => {
+		const [_, enhancedImg] = Object.entries(images).find(([key]) => project.image != null && project.image.endsWith(key)) ?? [];
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)